web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -385,15 +534,67 @@ export default class RunComparison extends ReactJS.Component {
               </Button>
             </div>
           </Description>
+          <Description term="Slected Division">
```

Comment:
Since we reference the selected components in the export modal, we can remove this data from the description component.

Reply:
Resolved